Resources/Prototypes/Entities/Mobs/NPCs/animals.yml

```diff
@@ -434,3 +434,102 @@
     - type: DamageStateVisualizer
       normal: penguin
       dead: penguin_dead
+
+- type: entity
+  name: mouse
+  parent: SimpleMobBase
+  id: MouseMob_Content
+  description: Squeak!
+  components:
+  - type: Tag
+    tags:
+    - Grindable
+  - type: GhostTakeoverAvailableComponent
+    makeSentient: true
+    name: mouse
+    description: A mouse in maint.
+  - type: Sprite
+    drawdepth: Mobs
+    layers:
+    - map: ["enum.DamageStateVisualLayers.Base"]
+      state: mouse-0
+      sprite: Mobs/Animals/mouse.rsi
+  - type: Clothing
+    QuickEquip: false
+    sprite: Mobs/Animals/mouse.rsi
+    HeldPrefix: 0
+    ClothingPrefix: 0
+    Slots:
+    - Helmet
+  - type: Physics
+    bodyType: Dynamic
```

Comment:
Do these work with Dynamic? I would've thought KinematicController unless I stuffed it.

Reply:
Tested, the KinematicControllers break them even more.